common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -948,6 +948,13 @@ def make_course_key(self, org, course, run):
         """
         return CourseLocator(org, course, run)
 
+    def make_course_usage_key(self, course_key):
+        """
+        Return a valid :class:`~opaque_keys.edx.keys.UsageKey` for this modulestore
+        that matches the supplied course_key.
+        """
+        return BlockUsageLocator(course_key, 'course', 'course')
```

Comment:
Ah, I didn't realize library keys were out of scope. LibraryLocators subclass CourseKey, so I thought maybe this was a general mechanism for going from those identifiers to their root nodes.

I think CCX would need to return `CCXBlockUsageLocator`, but I'm not sure. I was just curious if it was a known thing. Thank you.